catalog.go

```diff
@@ -0,0 +1,34 @@
+package gettext
+
+// Catalog of translations for a given locale.
+type Catalog struct {
+	mos []*mocatalog
+}
+
+func (c Catalog) findMsg(msgid string, usePlural bool, n uint32) (msgstr string, ok bool) {
```

Comment:
(nitpick) I'm not a fan of "usePlural bool" in functions like this. The problem is that with a bool it's really hard to know what is going on when just looking at the function call. E.g.:
```
c.findMsg(msgid, false, 0);
```
does not tell me what "false" is about unless I remember or look at the function signature.

Now of course here where it's all super close together it does not matter much :) Plus it exists already in mo.findMsg() etc. Just a pet-peeve of mine.